cypress/integration/ta/allow_or_disable_questions.spec.js

```diff
@@ -38,15 +38,30 @@ describe("Allow or disable new questions for a queue", () => {
 
 describe("When allow questions is disabled", () => {
   beforeEach(() => {
-    createAndLoginStudent();
+    createAndLoginTA();
+
     createQueue({
-      courseId: "student.course.id",
-      allowQuestions: false,
-      identifier: "queue",
+      courseId: "ta.course.id",
     });
+
+    checkInTA();
+
   });
+
   it("student cannot add new questions when new questions are disabled", function () {
+    cy.visit(`/course/${this.queue.courseId}/queue/${this.queue.id}`);
+    cy.get("[data-cy='editQueue']").click();
+    cy.get("[data-cy='allow-questions-toggle']").click();
```

Comment:
Out of curiosity, why did this change to manually setting the queue as disallowing questions? Looks like it was originally just creating the queue with `allowQuestions: false`

Reply:
the seeds were acting up, this is now working and not necessary